### ISSUE TITLE
As maintainer I want go sub test to run in parallel so that testing time is reduced improving velocity and reducing tests's costs

### ISSUE THREAD

**Issue by BrunoReboul** [BrunoReboul/ram#36]
aka generalize what is already done in  [this test](https://github.com/BrunoReboul/ram/blob/bruno/utilities/ramcli/func_getversions_unit_test.go#L55)
